coderd/httpapi/websocket.go

```diff
@@ -4,8 +4,9 @@ import (
 	"context"
 	"time"
 
-	"cdr.dev/slog"
 	"nhooyr.io/websocket"
+
+	"cdr.dev/slog"
```

Comment:
nit: is it expected or is the linter impaired?

Reply:
Might be my linter.